pull/github.go

```diff
@@ -308,13 +392,21 @@ func (ghc *GitHubContext) loadPagedData() error {
 			complete++
 		}
 
-		if complete == 2 {
+		for _, c := range q.Repository.Collaborators.Edges {
+			collaborators[c.Node.Login] = strings.ToLower(string(c.Permission))
```

Comment:
This is minor, but I think you can use `string` in the query struct definition and avoid this cast. The specific types are mostly defined for use as input data for mutation queries.